pkg/exchange/okex/query_order_trades_test.go

```diff
@@ -0,0 +1,43 @@
+package okex
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryOrderTrades(t *testing.T) {
+	key := os.Getenv("OKEX_API_KEY")
+	secret := os.Getenv("OKEX_API_SECRET")
+	passphrase := os.Getenv("OKEX_API_PASSPHRASE")
+	if len(key) == 0 && len(secret) == 0 {
+		t.Skip("api key/secret are not configured")
+		return
+	}
+	if len(passphrase) == 0 {
+		t.Skip("passphrase are not configured")
+		return
+	}
+
+	e := New(key, secret, passphrase)
+
+	queryOrder := types.OrderQuery{
+		OrderID: "609869603774656544",
+	}
+	transactionDetail, err := e.QueryOrderTrades(context.Background(), queryOrder)
```

Comment:
The test access okex api. That means you can't control how long the operation is. In this case, we often apply timeout to the context to ensure the unit test will be finished within a specific time. Otherwise, the execution time is not deterministic.

Reply:
updated